src/store/RootState.ts

```diff
@@ -5,6 +5,7 @@ export default interface RootState {
 	conditionRows: ConditionRow[];
 	errors: Error[];
 	limit?: number;
```

Comment:
It makes sense to make that field non-optional right away. We have to touch every test where this is relevant anyway to add the `useLimit` field, so we can add this as well.